Ubuntu-Downloading-Text-Editors-and-IDEs.md

```diff
@@ -62,13 +64,13 @@ You can now drag this file to the launcher, enabling you to start Eclipse.
 
 **Installing Intellij IDEA:**
 
-Download the latest version of Intellij IDEA: https://www.jetbrains.com/idea/download/#section=linux
+Download the latest version of Intellij IDEA: [Intellij](https://www.jetbrains.com/idea/download/#section=linux)
```

Comment:
Download the latest version of [**Intellij IDEA**](https://www.jetbrains.com/idea/download)

```diff
- Download the latest version of Intellij IDEA: [Intellij](https://www.jetbrains.com/idea/download/#section=linux)
+ Download the latest version of [**Intellij IDEA**](https://www.jetbrains.com/idea/download)
```